.travis.yml

```diff
@@ -8,3 +8,4 @@ before_install:
 
 script:
   - godep go test  ./pkg/... -race -cover
+  - godep go test  ./cmd/agent -test.run TestDir
```

Comment:
travis may support zk directly, if not, we can install it using apt-get or download it, untar it and run by ourself.